gamemode/modules/base/sv_data.lua

```diff
@@ -236,6 +236,16 @@ function DarkRP.retrieveRPNames(name, callback)
     end)
 end
 
+function DarkRP.offlinePlayerData(steamid, callback, failed)
+    steamid = steamid:upper()
+    if !steamid:find( "^STEAM_%d:%d:%d+$" ) then failed() return end
+    local uniqueid = util.CRC( "gm_" .. steamid .. "_gm" )
```

Comment:
Nice consistency.

To be consistent with the rest of the codebase, use `not` instead of `!` and don't put spaces just before and after the parenthesis/brackets/whatever.